lib/sequelize.js

```diff
@@ -28,6 +28,7 @@ const _ = require('lodash');
  * In addition to sequelize, the connection library for the dialect you want to use should also be installed in your project. You don't need to import it however, as sequelize will take care of that.
  *
  * @class Sequelize
+ * @lands Errors
```

Comment:
I think what you meant was `@lends` http://usejsdoc.org/tags-lends.html
But I don't think it's the right tag for what you intended, it should be `@mixes Errors` http://usejsdoc.org/tags-mixes.html